tools/release/publish_release.sh

```diff
@@ -628,6 +628,8 @@ start_ver_tag=fleet-$start_version
 
 if [[ "$minor" == "true" ]]; then
     echo "Minor release from $start_version to $next_ver"
+    # For scheduled minor releases, we want to branch off of main
+    start_ver_tag="main"
```

Comment:
I think this is the most straightforward fix for now. We do need to refactor since our process has changed since you first created this.